loader/plugins/prune.js

```diff
@@ -0,0 +1,42 @@
+const CordlrPlugin = require('cordlr-plugin')
+
+module.exports = class HelpPlugin extends CordlrPlugin {
+  constructor (bot, config) {
+    super(bot, config)
+
+    this.name = 'Cordlr Core - Prune'
+    this.description = 'Cordlr Core plugin for pruning an specific amount of messages'
+
+    this.commands = {
+      'prune': {
+        'usage': '<x>',
+        'function': 'pruneMessages',
+        'description': 'Prunes the last x messages',
+        'permissions': [
+          'MANAGE_MESSAGES'

```

Comment:
Did you try give your bot the "Administrator" permission? I don't know if this is bad practice, but it should work. I'll implement a promise handler for the rejection.